elastiknn-testing/docker-compose.yml

```diff
@@ -44,7 +44,9 @@ services:
       - discovery.zen.ping.unicast.hosts=elasticsearch_master
       - node.master=false
       - bootstrap.memory_lock=true
-      - ES_JAVA_OPTS=-Xms4G -Xmx4G
+      - ES_JAVA_OPTS=-Xms4G -Xmx4G -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.port=8097 -Dcom.sun.management.jmxremote.rmi.port=8097 -Djava.rmi.server.hostname=localhost
```

Comment:
Added JMX connection settings and exposed port to allow attaching VisualVM to the testing container.